index.html

```diff
@@ -40,4 +40,234 @@
+          <div class="frame-2">
+            <div class="Client">
+              <h2 class="CANPOY">CANPOY</h2>
+              <i class="counter fa-solid fa-circle"></i>
+              <div class="Role"><h2 class="back-end">Back End Dev</h2></div>
+              <i class="counter fa-solid fa-circle"></i>
+              <div class="Role"><h2 class="back-end">2022</h2></div>
+            </div>
+          </div>
+        </div>
+        <div class="information">
+          <p class="des-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        
+        <ul class="tags">
+          <li class="tag"> <a class="html" href="">html</a></li>
+          <li class="tag"> <a class="html" href="">CSS</a></li>
+          <li class="tagj"> <a class="html" href="">JavaScript</a></li>
+        </ul>
+        
+        <div class="action">
+         <button class="normal" type="button">See Project</button>
+        </div>
+        
+      </div>
+         
+        
+      
+    </div>
+    
+
+  </section>
+
+  <section class="section-5">
+    <div class="card">
+      <div class="photo">
+        <img class="work-img" src="Microverse-Student-Project-1/img5.png" alt="photo">
+      </div>
+      <div class="left-block">
+        <div class="primary-text">
+          <h1 class="project-title">Multi-Post
+            Stories</h1>
+          <div class="frame-2">
+            <div class="Client">
+              <h2 class="CANPOY">CANPOY</h2>
+              <i class="counter fa-solid fa-circle"></i>
+              <div class="Role"><h2 class="back-end">Back End Dev</h2></div>
+              <i class="counter fa-solid fa-circle"></i>
+              <div class="Role"><h2 class="back-end">2022</h2></div>
+            </div>
+          </div>
+        </div>
+        <div class="information">
+          <p class="des-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        
+        <ul class="tags">
+          <li class="tag"> <a class="html" href="">html</a></li>
+          <li class="tag"> <a class="html" href="">CSS</a></li>
+          <li class="tagj"> <a class="html" href="">JavaScript</a></li>
+        </ul>
+        
+        <div class="action">
+         <button class="normal" type="button">See Project</button>
+        </div>
+        
+      </div>
+         
+        
+      
+    </div>
+    
+
+  </section>
+
+  <section class="section-6">
+    <div class="img-holder">
+      <img class="about-my-self" src="Microverse-Student-Project-1/img6.png" alt="">
+    </div>
+      <div class="top">
+        <h1 class="first-text">About <br><span>Myself</span></h1>
+        <h1 class="second-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h1>
```

Comment:
- [ ] I would suggest you use a `<p>` as this is just a paragraph and not a title or a subtitle. Kindly, apply this change for semantic reasons.